api/app/Enums/ArmedForcesStatus.php

```diff
@@ -2,9 +2,18 @@
 
 namespace App\Enums;
 
+use App\Traits\HasLocalization;
+
 enum ArmedForcesStatus
 {
+    use HasLocalization;
+
     case VETERAN;
     case MEMBER;
     case NON_CAF;
+
+    public static function getLangFilename(): string
+    {
+        return 'armed_forces_status';
```

Comment:
Missing file?

Reply:
Added in https://github.com/GCTC-NTGC/gc-digital-talent/pull/10701/commits/e0143f0d1256e92bf43756ccdcb5397bdc95b781